### DISCUSSION TITLE
Way to get rid of InsecureRequestWarning with private index?

### DISCUSSION THREAD

**Discussion by genericmoniker** [pdm-project/pdm#1553] [Q&A] [answered]
I've configured my pyproject.toml to use a private index that has a self-signed certificate, like so:

```
[[tool.pdm.source]]
url = "https://my-private-index"
verify_ssl = false
name = "pypi"
```
It works fine, but now most pdm commands I run spew many lines of: 

InsecureRequestWarning: Unverified HTTPS request is being made to host 
'my-private-index'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings

I know there are ways of suppressing those within my own code, but is there a way to suppress them while using pdm? Is there a chance pdm would do the suppression itself given the explicit `verify_ssl = false`?

**Comment by frostming** [comment, marked as answer]
`unearth` [0.7.0](https://github.com/frostming/unearth/releases/tag/0.7.0) has been released which suppresses the warning. You may need to update the dependency.
